### DISCUSSION TITLE
IFS 0.4 retirement, CAMS as replacement?

### DISCUSSION THREAD

**Discussion by phillipseamore** [open-meteo/open-meteo#751] [General]
Since CAMS has lifted the delay on met params I'm wondering if they should be used to replace the IFS 0.4 forecast. It does include many more met parameters than IFS 0.4 and 0.25 so would add value.

**Comment by patrick-zippenfenig** [comment]
Yes, it would be an option. I just checked the FTP distribution of CAMS global and they added additional fields like temperature 2m, dew point, 10 m wind, pressure in 1-hourly resolution. Model and pressure level in 3-hourly intervals (no cloud fraction, but specific humidity Q). Delay to initialisation 6:50. However, only 0z and 12z runs are available, only 120 forecast hours and no precipitation, soil, solar, 100m wind data...

Given those limitations, do you think it is useful to add it?

Reference [CAMS ADS](https://ads.atmosphere.copernicus.eu/cdsapp#!/dataset/cams-global-atmospheric-composition-forecasts?tab=form)

**Comment by phillipseamore** [comment]
Precip, CAPE, CIN, max gust last 3h, visiblity, radiation and more are under the "slow-access" section and was what got my attention. As I've probably stated before I'm mostly interested in what might be an issue or danger in the next 1-3 days so the 120h forecast doesn't bother me, I didn't notice the 7h delay though the quality of ECMWF forecasts might make up for it (and even with the limited resolution).

Let's see if we have input from some more users.

**Reply by patrick-zippenfenig** [reply]
"Slow-access" is usually not an understatement. Currently CAMS Global forecasts are access through their FTP server instead of the Atmospheric Data Store ADS. The FTP distribution does not include variables from the "Slow-access" category. Data from ADS has a higher latency and it takes longer to query data through their job queue system. Latencies of 10 or more hours are likely....
